lib/dom.js

```diff
@@ -1060,6 +1068,167 @@ Node.prototype = {
 	 * @type {string | null}
 	 */
 	localName: null,
+	/**
+	 * The URI is currently always `about:blank` since that's what happens when you create a
+	 * document from scratch.
+	 * This could be improved by making it possible to add a baseURI to the document when creating
+	 * one so it's childNodes just use it's URI instead.
+	 *
+	 * @type {string}
+	 */
+	baseURI: 'about:blank',
+	/**
+	 * Is true if this node is part of a document.
+	 *
+	 * @type {boolean}
+	 */
+	get isConnected() {
+		var rootNode = this.getRootNode();
+		if (rootNode && rootNode.nodeType === rootNode.DOCUMENT_NODE) {
+			return true;
+		}
+
+		return false;
+	},
+	/**
+	 * Checks whether or not `other` is an inclusive descendant of this node.
+	 *
+	 * @param {Node} [other]
+	 * The node to check.
+	 * @returns {boolean}
+	 * True if `other` is an inclusive descendant of this node; false otherwise.
+	 * @see https://dom.spec.whatwg.org/#dom-node-contains
+	 */
+	contains: function (other) {
+		if (!other) {
+			return false;
+		}
```

Comment:
I would suggest we add a test that passes `undefined` or `null` to the function to cover the condition.
It makes sense to have it, even though the types say you can only pass a `Node` which can never be falsy.

As long as our tests are not type checked, it is possible to cover the case :)